paper/paper.md

```diff
@@ -16,48 +16,201 @@ authors:
+#> (Intercept)         |     0.13 | 0.26 | [-0.37, 0.64] |  0.52 | 0.606
+#> treatment [control] |    -0.27 | 0.37 | [-0.99, 0.45] | -0.73 | 0.466
+#> 
+#> # Random Effects
+#> 
+#> Parameter          | Coefficient |           95% CI
+#> ---------------------------------------------------
+#> SD (Intercept: ID) |        0.05 | [0.00, 9.86e+28]
+```
+
+Then, the performance of the model can be assessed with the `{performance}` package:
+
+
+``` r
+model_performance(
+  model,
+  metrics = c("AIC", "BIC", "R2")
+)
+#> # Indices of model performance
+#> 
+#> AIC   |   BIC | R2 (cond.) | R2 (marg.)
+#> ---------------------------------------
+#> 171.8 | 180.2 |      0.006 |      0.005
+```
+
+The results can be visualized using the `{see}` package by, for example, plotting the model's predictions from `{modelbased}` (Figure 1):
+
+
+``` r
+predictions <- estimate_means(model, "treatment")
+plot(predictions) + theme_modern(show.ticks = TRUE) # add nice theme
+```
+
+\begin{figure}
+\includegraphics[width=1\linewidth]{paper_files/figure-latex/fig1-1} \caption{Predicted probability of alertness by treatment group.}\label{fig:fig1}
+\end{figure}
+
+Finally, a full report of the analysis can be generated with the `{report}` package:
+
+
+
+
+``` r
+report(model)
+#> We fitted a logistic mixed model (estimated using ML and nlminb
+#> optimizer) to predict alertness with treatment (formula: alertness ~
+#> treatment). The model included ID as random effect (formula: ~1 |
+#> ID). The model's total explanatory power is very weak (conditional
+#> R2 = 6.35e-03) and the part related to the fixed effects alone
+#> (marginal R2) is of 5.44e-03. The model's intercept, corresponding
+#> to treatment = coffee, is at 0.13 (95% CI [-0.37, 0.64], p = 0.606).
+#> Within this model:
+#> 
+#>   - The effect of treatment [control] is statistically non-significant
+#> and negative (beta = -0.27, 95% CI [-0.99, 0.45], p = 0.466)
+#>   - The effect of treatment [control] is statistically non-significant
+#> and negative (beta = -0.27, 95% CI [-0.99, 0.45], p = 0.466)
+#>   - The effect of treatment [control] is statistically non-significant
+#> and negative (beta = -0.27, 95% CI [-0.99, 0.45], p = 0.466)
+#>   - The effect of treatment [control] is statistically non-significant
+#> and negative (beta = -0.27, 95% CI [-0.99, 0.45], p = 0.466)
+#>   - The effect of treatment [control] is statistically non-significant
+#> and negative (beta = -0.27, 95% CI [-0.99, 0.45], p = 0.466)
+#> 
+#> Standardized parameters were obtained by fitting the model on a
+#> standardized version of the dataset. 95% Confidence Intervals (CIs)
+#> and p-values were computed using a Wald z-distribution
+#> approximation.
+```
+
+The `model_dashboard()` function generates an interactive model summary, integrating several **easystats** steps into one command. Since this paper is a static document, we cannot show a live demonstration, but Figure 2 provides a visual example.
```

Comment:
```suggestion
The `model_dashboard()` function generates an interactive model summary, integrating several **easystats** steps into a single command. Since this paper is a static document, Figure 2 provides a screenshot of the interactive report.
```